app/models/user.rb

```diff
@@ -133,7 +133,7 @@ def guessed_last_name
   end
 
   def casual_name
-    first_name.present? ? first_name : username
+    first_name
```

Comment:
Might need to leave this as is -- legacy users can have blank first names but they must have non-blank usernames.  And we need some way to say "Hi <name>" to them.  Can discuss if you want.